applications/luci-app-ssr-plus/root/usr/share/shadowsocksr/subscribe.lua

```diff
@@ -202,7 +206,8 @@ local function processData(szType, content)
 		end
 		if info.tls == "tls" or info.tls == "1" then
 			result.tls = "1"
-			result.tls_host = info.host
+			if info.host then result.tls_host = info.host end
+			if info.sni then result.tls_host = info.sni end
```

Comment:
我倾向于使用 if ... elseif ... end
```suggestion
			if info.sni then
				result.tls_host = info.sni
			elseif info.host then
				result.tls_host = info.host
			end
```